### DISCUSSION TITLE
was i using the resolves field in my PRs wrong

### DISCUSSION THREAD

**Discussion by markverb1** [FlagClicked/FlagClicked#54] [Help] [answered]
read the title
was i suppost to mention an issue
i'm still a GH newbie lol
looking at @TheColaber's PRs i think i was using it wrong

**Comment by danielgallab** [comment, marked as answer]
Not sure what you mean... what resolves field? If you are talking about the prettier action, don't worry if that action fails.

**Reply by markverb1** [reply]
![IMG_20210505_171958](https://user-images.githubusercontent.com/77983517/117156282-2f157f80-adc6-11eb-978f-a9afaac91fdf.jpg)
this

**Reply by danielgallab** [reply]
Oh... when you type `Resolves #XX` on your PR, it will automatically close that issue when your PR gets merged.

**Reply by markverb1** [reply]
> Oh... when you type `Resolves #XX` on your PR, it will automatically close that issue when your PR gets merged.

oh so then i was using it wrong.
